### ISSUE TITLE
GPU Support?

### ISSUE THREAD

**Issue by smk508** [refresh-bio/FAMSA#20]
I was wondering why GPU support was removed in the recent versions of FAMSA and if there are any caveats to using an older version. I am trying to do a very large multiple-sequence alignment, and I have a lot of GPU resource, so I'm wondering if it makes sense to use an older version of FAMSA or to just do it on CPU.

Thanks

**Comment by agudys** [comment]
Hello,

Since only the first stage of FAMSA was subject to the GPU parallelization, the speedups were not that impressive (you can see in the paper, that 8-12 CPU cores provided similar performance as Quadro with 3072 cores). Therefore, due to portability issues and the fact that FAMSA in CPU mode is still very fast, we decided to drop the support of the GPU mode in the latest revisions. 

However, if regular FAMSA is too slow for your needs, please take a look on the `experimental` branch. We provide there `-medoidtree` feature for constructing very fast approximated trees. This allows calculating huge alignments in short time (e.g., 3M sequences required 2 hours while default FAMSA needed a day). 

To sum up, we recommend sticking to the CPU revision.

Regards,
Adam
